website/content/api-docs/api-structure.mdx

```diff
@@ -40,9 +40,9 @@ $ curl \
     http://127.0.0.1:8500/v1/agent/members
 ```
 
-Previously this was provided via a `?token=` query parameter. This functionality
-exists on many endpoints for backwards compatibility, but its use is **highly
-discouraged**, since it can show up in access logs as part of the URL.
+Though the token could also be provided via a `?token=` query parameter,
+that method is **highly discouraged** because the token can show up in access logs as part of the URL.
+The `?token=` query parameter is deprecated and will be removed in Consul 1.17.
```

Comment:
```suggestion
~> **Security Note:** Though you could pass the token through a `?token=` query parameter, this method is highly discouraged because the token can show up in access logs as part of the URL. The `?token=` query parameter is deprecated and will be removed in Consul 1.17.
```